src/Features/ERDDAP/Platform/Forecasts/Page/index.tsx

```diff
@@ -1,43 +1,116 @@
-import React from "react"
-import { connect } from "react-redux"
+/**
+ * Load and display forecasts
+ */
+import { Point } from "@turf/helpers"
+import * as React from "react"
+import { Alert, Row, Col } from "reactstrap"
 
 import { MultipleLargeTimeSeriesChart } from "components/Charts"
-import { StoreState } from "Shared/constants/store"
 import { round } from "Shared/math"
 import { DataTimeSeries } from "Shared/timeSeries"
 import { UnitSystem } from "Features/Units/types"
 import { converter } from "Features/Units/Converter"
 
-import { ForecastSource, PlatformFeatureWithDatasets } from "../../../types"
-import { ErddapDatasetLoader, ErddapDatasetStatus } from "../../Dataset"
-import { ForecastLoader } from "../Loader"
+import { useDataset, useForecast, useForecasts } from "../../../hooks"
+import { ForecastSource, PlatformFeature, PlatformTimeSeries } from "../../../types"
 
-export interface Props {
-  platform: PlatformFeatureWithDatasets
-  type: string
+interface Props {
+  platform: PlatformFeature
+  forecast_type: string
```

Comment:
Codacy found an issue: [Identifier 'forecast_type' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)